cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -399,51 +390,6 @@ var (
 	imageDigestMatcher = regexp.MustCompile(`^.*(?i:sha256:([[:alnum:]]{64}))$`)
```

Comment:
This is not used anymore. Can be removed

Reply:
thanks for catching that. updated.